### DISCUSSION TITLE
Static Analysis Support

### DISCUSSION THREAD

**Discussion by gauravmak** [spatie/laravel-activitylog#950] [Ideas] [answered]
We are facing the following issues while running Larastan on our codebase.

![Image_9 jpg](https://user-images.githubusercontent.com/11887260/131794998-4d9f395c-6698-4ebd-bbf9-75b4d451b894.png)

One of the ways to fix is to add properties of the model in the doc blocks of the model. Are collaborators open for this?

**Comment by Gummibeer** [comment, marked as answer]
Hey,
yes - I would merge a PR adding a full doc-block as the IDE-Helper generates. It should use FQCN instead of imports and also use `Carbon\Carbon` instead of `Illuminate\Support\Carbon` and the last `@extends` or `@mixin` line can be removed as well as all the magic `where` methods.
